chips/sam4l/src/usbc/register_macros.rs

```diff
@@ -0,0 +1,65 @@
+//! Macros for defining USBC registers
+
+#[macro_export]
+macro_rules! reg {
+    [ $offset:expr, $description:expr, $name:ident, "RW" ] => {
+        #[allow(dead_code)]
+        pub const $name: Reg<u32> = Reg::new((USBC_BASE + $offset) as *mut u32);
```

Comment:
`Reg` et al should be fully qualified here (that's generally true in macros) otherwise it would require the user of the macro to have it in scope already.